public/app/features/dashboard-scene/serialization/transformSceneToSaveModelSchemaV2.test.ts

```diff
@@ -1065,18 +1108,6 @@ function createAnnotationLayers() {
       isEnabled: true,
       isHidden: true,
     }),
-    // this could happen if a dahboard was created from code and the datasource was not defined
-    new DashboardAnnotationsDataLayer({
-      key: 'layer3',
-      query: {
-        name: 'query3',
-        enable: true,
-        iconColor: 'green',
-      },
-      name: 'layer3',
-      isEnabled: true,
-      isHidden: true,
-    }),
```

Comment:
@ivanortegaalba @axelavargas  - removed this. With the current implementation it's not possile to config annotations data layer without data source.